renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -142,6 +155,23 @@ struct PixelHistoryShaderCache
     return m_FixedColFS;
   }
 
+  VkShaderModule GetPrimitiveIdShader()
```

Comment:
The vulkan shader cache already creates a `VkShaderModule` for each builtin shader. Since there's no customisation here (i.e. patching the SPIR-V or anything) could you use `GetBuiltinModule` instead of `GetBuiltinBlob`?